server/src/internalClusterTest/java/org/elasticsearch/search/suggest/CompletionSuggestSearchIT.java

```diff
@@ -1590,15 +1586,10 @@ public void testCompletionWithCollapse() throws Exception {
     }
 
     public static boolean isReservedChar(char c) {
-        switch (c) {
-            case '\u001F':
-            case TokenStreamToAutomaton.HOLE:
-            case 0x0:
-            case ContextSuggestField.CONTEXT_SEPARATOR:
-                return true;
-            default:
-                return false;
-        }
+        return switch (c) {
+            case '\u001F', TokenStreamToAutomaton.HOLE, 0x0, ContextSuggestField.CONTEXT_SEPARATOR -> true;
```

Comment:
I think this is another example where it was clearer before, but I'll leave it up to you.